src/components/Video/style.css

```diff
@@ -102,33 +106,36 @@
   text-align: center;
   position: absolute;
   z-index: 1;
-  top: 64px;
+  top: 64*@unit;
   left: 0;
   right: 0;
 }
 
 .recordingIndicator {
-  font-size: 11px;
   text-align: center;
-  line-height: 16px;
-  margin-bottom: 16px;
+  margin-bottom: 16*@unit;
 
   /* Red circle */
   &::before {
     content: '';
     display: inline-block;
-    width: 8px;
-    height: 8px;
-    margin-right: 5px;
+    width: 8*@unit;
+    height: 8*@unit;
+    margin-right: 5*@unit;
     background: #DC2A2A;
-    border-radius: 4px;
+    border-radius: 4*@unit;
     line-height: 120%;
   }
 }
+.recordingIndicatorText {
+  font-size: 11*@unit;
+  line-height: 1.45;
```

Comment:
curious: this seems to be tantalisingly close to `line-height: 1.5`, what is the reason?

Reply:
Just keeping the design pixel-perfect to the SDK as it stands now, where possible

Reply:
I guess, you figured out what was the defacto lineheight when it was all still `px`?